app/server/models/user.py

```diff
@@ -56,6 +56,11 @@
 )
 
 
+class RegistrationMethodEnum(Enum):
```

Comment:
Good, and as mentioned the other enums that sempo use seem to be defined in `app/server/utils` instead. let's do the same.